product_docs/docs/epas/14/epas_compat_cat_views/10_all_indexes.mdx

```diff
@@ -9,25 +9,25 @@ legacyRedirectsGenerated:
 | `index_type`      | `TEXT`                  | The index type is always `BTREE`. Included for compatibility only.                                   |
 | `table_owner`     | `TEXT`                  | User name of the owner of the indexed table.                                                         |
-| `table_name`      | `TEXT`                  | The name of the indexed table.                                                                       |
+| `table_name`      | `TEXT`                  | Name of the indexed table.                                                                       |
 | `table_type`      | `TEXT`                  | Included for compatibility only. Always set to `TABLE`.                                              |
 | `uniqueness`      | `TEXT`                  | Indicates if the index is `UNIQUE` or `NONUNIQUE`.                                                   |
-| `compression`     | `CHARACTER(1)`          | Always set to `N` (not compressed). Included for compatibility only.                                 |
-| `tablespace_name` | `TEXT`                  | Name of the tablespace in which the table resides if other than the default tablespace.              |
+| `compression`     | `CHARACTER(1)`          | Always set to `N` (not compressed). Included only for compatibility.                                 |
+| `tablespace_name` | `TEXT`                  | Name of the tablespace in which the table resides if not the default tablespace.              |
 | `degree`          | `CHARACTER VARYING(10)` | Number of threads per instance to scan the index.                                                    |
-| `logging`         | `TEXT`                  | Always set to `LOGGING`. Included for compatibility only.                                            |
-| `status`          | `TEXT`                  | Included for compatibility only; always set to `VALID`.                                              |
-| `partitioned`     | `CHARACTER(3)`          | Indicates that the index is partitioned. Currently, always set to `NO`.                              |
-| `temporary`       | `CHARACTER(1)`          | Indicates that an index is on a temporary table. Always set to `N`; included for compatibility only. |
-| `secondary`       | `CHARACTER(1)`          | Included for compatibility only. Always set to `N`.                                                  |
-| `join_index`      | `CHARACTER(3)`          | Included for compatibility only. Always set to `NO`.                                                 |
-| `dropped`         | `CHARACTER(3)`          | Included for compatibility only. Always set to `NO`.                                                 |
+| `logging`         | `TEXT`                  | Always set to `LOGGING`. Included only for compatibility.                                            |
```

Comment:
Can we reverse the order of both sentences in the description column here to match with the rest of the description.

Reply:
Fixed